stock_full_location_reservation/security/groups.xml

```diff
@@ -0,0 +1,6 @@
+<odoo>
+    <record model="res.groups" id="group_user">
```

Comment:
```suggestion
    <record model="res.groups" id="stock_full_reservation">
```

Reply:
In other addons you have to use stock_full_location_reservation.group_user as xml_id.
I think this is good.